scripts/base/protocols/dhcp/main.zeek

```diff
@@ -112,6 +112,11 @@ export {
 	## Event that can be handled to access the DHCP
 	## record as it is sent on to the logging framework.
 	global log_dhcp: event(rec: Info);
+
+	# 67/udp is the server's port, 68/udp the client.
+	# 4011/udp seems to be some proxyDHCP thing.
+	option ports = { 67/udp, 68/udp, 4011/udp };
+	redef likely_server_ports += { 67/udp };
```

Comment:
Seems that's the only one where we can't use ports. How about splitting into `client_ports` and `server_ports` just in case .